### PR TITLE
docs: add "Installation", "Documentation" and "Usage" to "README.md"

### PR DESCRIPTION
**Pull request by graczhual**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-2bgyr83 SDK 文档编写 —— README.md](https://app.clickup.com/t/2bgyr83)

File: README.md [hunk 1]
```diff
@@ -7,5 +7,229 @@
...
+]
+```
+
+Checkout commit/branch/tag:
+
+```python
+>>> dataset.checkout("47293b32f28c4008bc0f25b847b97d6f")  # commit id
+>>> dataset.HEAD
+Commit("47293b32f28c4008bc0f25b847b97d6f")(
+  (parent_commit_id): '00000000000000000000000000000000',
```
Comment (Lee-000) [inline README.md:122]: Which root commit id should we use here, zeros or None?

File: README.md [hunk 2]
```diff
@@ -7,5 +7,229 @@
...
+>>> df.loc[0]["box2ds"]
+   xmin  ymin  xmax  ymax  category  attribute
+                                     difficult  occluded
+0  1.0   1.0   4.0   5.0   boat      False      False
+```
+
+```python
+>>> df["box2ds"][0]
+    xmin  ymin  xmax  ymax  category  attribute
+                                     difficult  occluded
```
Comment (AChenQ) [inline README.md:233]: Not alignment here

File: README.md [hunk 3]
```diff
@@ -7,5 +7,226 @@
 [![PyPI - Python Version](https://img.shields.io/pypi/pyversions/graviti)](https://pypi.org/project/graviti/)
 [![Downloads](https://pepy.tech/badge/graviti/month)](https://pepy.tech/project/graviti)
 
-Graviti Python SDK is a python library to access Graviti Data Platform and manage your datasets.
-It provides a pythonic way to access your datasets by Graviti OpenAPI.
+Graviti Python SDK is a python library to access [Graviti](https://www.graviti.com) workspace and
```
Comment (linjiX) [inline README.md:10]: The content `README.md` is similar to quick start in sphinx doc, is it possible to reuse the rst or md file?

**Review by AChenQ — APPROVED** [review]

**Review by Lee-000 — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (graczhual) [reply README.md:10]: The two files are not exactly the same, `README.md` has something more. Secondly, I haven't found a good way to use `README.md` in `docs/source/index`. Maybe we can optimize it later.

**Review by linjiX — APPROVED** [review]

**Pull request merged**